app/routes/agenda-items/index.ts

```diff
@@ -1,172 +1,18 @@
-}
-
-const agendaItemsQuery = ({
-  page,
-  keyword,
-  locationIds,
-  plannedStartMin,
-  plannedStartMax,
-}: {
-  page: number;
-  keyword?: string;
-  locationIds?: string;
-  plannedStartMin?: string;
-  plannedStartMax?: string;
-}) => ({
-  include: [
-    'sessions.governing-body.is-time-specialization-of.administrative-unit.location',
-    'sessions.governing-body.administrative-unit.location',
-  ].join(','),
-  sort: '-sessions.planned-start',
-  filter: {
-    sessions: {
-      ':gt:planned-start': plannedStartMin,
-      ':lt:planned-start': plannedStartMax,
-      'governing-body': {
-        'is-time-specialization-of': {
-          'administrative-unit': {
-            location: {
-              ':id:': locationIds,
-            },
-          },
-        },
-      },
-    },
-    ':or:': {
-      title: keyword,
-      description: keyword,
-    },
-  },
-  page: {
-    number: page,
-    size: 10,
-  },
-});
-
-export async function getAgendaItems(
-  context: AgendaItemsIndexRoute | AgendaItemsIndexController,
-  params: AgendaItemsParams,
-  currentPage = 0
-) {
-  const controller =
-    context instanceof AgendaItemsIndexController
-      ? context
-      : context.routeController;
-
-  /**
-   * The || undefined is important!
-   *
-   * Some queryParams are set to '' in the controller this is for
-   * the sake of having empty values not leaving behind a ?queryparam=&...
-   *
-   * However, whether it be because of the way we build our queries,
-   * because of our back-ends code, or because of internal Ember-Data structure,
-   * it does not like being given '' when you intend to disable that filter
-   * So this ensures that '' as well as undefined get both resolved to undefined!
-   */
-  controller.keyword = params.keyword || undefined;
-  controller.municipalityLabels = params.municipalityLabels || undefined;
-  controller.plannedStartMin = params.plannedStartMin || undefined;
-  controller.plannedStartMax = params.plannedStartMax || undefined;
-
-  const locationIds = await context.municipalityList.getLocationIdsFromLabels(
-    controller.municipalityLabels
-  );
-
-  const agendaItems: AdapterPopulatedRecordArrayWithMeta<AgendaItemModel> =
-    await context.store.query(
-      'agenda-item',
-      agendaItemsQuery({
-        page: currentPage,
-
-        locationIds: locationIds,
-
-        keyword: controller.keyword,
-        plannedStartMin: controller.plannedStartMin,
-        plannedStartMax: controller.plannedStartMax,
-      })
-    );
-
-  if (context instanceof AgendaItemsIndexRoute) {
-    controller.set('agendaItems', agendaItems.slice());
-  }
-
-  controller.set('currentPage', currentPage);
-  controller.set('count', getCount(agendaItems));
-
-  return agendaItems.slice();
-}
-
-export default class AgendaItemsIndexRoute extends Route {
-  @service declare store: Store;
-  @service declare municipalityList: MunicipalityListService;
 
+export default class AgendaItemsRoute extends Route {
```

Comment:
Whoops, seems I made a mistake during the merge conflict resolving. Shouldn't have been renamed back.